src/Symfony/Component/BrowserKit/Tests/AbstractBrowserTest.php

```diff
@@ -159,17 +104,6 @@ public function testGetResponseNull()
         $this->assertNull($client->getResponse());
     }
 
-    public function testGetInternalResponse()
```

Comment:
from #30602, this test is wrong: it relies on a class that extends `Response` while the class is `@final`
from #31674, it has already been removed from master, despite the fact it was not marked as legacy.
That's two mistakes in a row, fortunately, the recent improvements of DebugClassLoader will allow catching those before they happen :)